src/main/java/tempest/GUI/DeleteModulePage.java

```diff
@@ -0,0 +1,102 @@
+package tempest.GUI;
+
+import java.awt.event.ActionEvent;
+import java.util.Objects;
+
+import javax.swing.BoxLayout;
+import javax.swing.JButton;
+import javax.swing.JComboBox;
+import javax.swing.JPanel;
+
+import tempest.State;
+import tempest.GUI.components.BackButton;
+import tempest.GUI.components.ModuleDropDown;
+
+public class DeleteModulePage extends Page {
+    private final State state;
+    private final GUIManager manager;
+    private final ModuleDropDown moduleDropDown = new ModuleDropDown();
+
+    private JPanel deleteModulePanel;
+    private JPanel buttonPanel;
+    private JPanel dropDownPanel;
```

Comment:
IntelliJ is shouting at me that these 3 can be locals. I doubt we need to get these panels so no reason not to have them be local.